flasksrc/quiz.py

```diff
@@ -58,3 +85,96 @@ def get(self, quiz_id):
                 )
 
         return {'items': result}, 200
+
+
+class ToyNetQuizScore(MethodResource):
+    @jwt_required()
+    @use_kwargs(ToyNetQuizScorePostReq)
+    @marshal_with(ToyNetQuizScorePostResp)
+    def post(self, **kwargs):
+        try:
+            req = ToyNetQuizScorePostReq().load(kwargs)
+        except ValidationError as e:
+            abort(400, message=f'malformed create submission request: {e.messages}')
+
+        quiz_id = req['quiz_id']
+        user_id = req['user_id']
+        count_correct = req['count_correct']
+        count_wrong = req['count_wrong']
+
+        db = get_db()
+        try:
+            db.execute(
+                'INSERT INTO toynet_quiz_scores(quiz_id,user_id,count_correct,count_wrong)'
+                ' VALUES((?), (?), (?), (?))',
+                (quiz_id, user_id, count_correct, count_wrong,)
+            )
+            db.commit()
+            rows = db.execute(
+                'SELECT scores.submission_id'
+                ' FROM toynet_quiz_scores AS scores'
+                ' WHERE scores.quiz_id = (?) AND scores.user_id = (?)',
```

Comment:
You might want to `ORDER` this `BY`  `submitted` (don't know if it would be `ACSD` or `DESC`) or `submission_id` to make sure you get the LAST id.